scripts/upgrade

```diff
@@ -55,12 +65,12 @@ ynh_clean_setup () {
 ynh_abort_if_errors
 
 #=================================================
-# ENSURE DOWNWARD COMPATIBILITY
+# MANAGE UPGRADE
 #=================================================
-ynh_print_info "Ensuring downward compatibility..."
+ynh_print_info --message="Ensuring downward compatibility..."
```

Comment:
Hmokay but we already "Ensured downward compat" earlier ...? (same message a few blocks before)

(That's mostly cosmetic but still can create confusion for people reading those)

Reply:
Changed the message for `Managing upgrade from previous version...`